src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts

```diff
@@ -291,6 +291,43 @@ describe('chartOptionsBuilder', () => {
                 });
             });
         });
+
+        describe('optional stacking with viewBy 2 attributes', () => {
+            const chartOptions = {
+                isViewByTwoAttributes: true,
+                type: 'column'
+            };
+
+            it('should validate with "dataTooLarge: false" with category number less than default limit', () => {
+                const data = {
+                    series: Array(1),
+                    categories: Array(3).fill({
+                        name: 'Month',
+                        categories: Array(31)
+                    })
+                };
+                const validationResult = validateData(undefined, { ...chartOptions, data });
+                expect(validationResult).toEqual({
+                    dataTooLarge: false,
+                    hasNegativeValue: false
+                });
+            });
+
+            it('should validate with "dataTooLarge: true" with category number greater than default limit', () => {
```

Comment:
Please use data driven tests for both of tests
https://confluence.intgdc.com/display/~dan.homola/Testing+best+practices+and+guidelines

Reply:
it's done :)